### ISSUE TITLE
Tasks left to do

### ISSUE THREAD

**Issue by KonradJanica** [KonradJanica/CG_Assign3#4]
High priority IMO (no particular ordering):
- G̶a̶m̶e̶C̶o̶n̶t̶r̶o̶l̶l̶e̶r̶ ̶r̶e̶d̶e̶s̶i̶g̶n̶?̶
- ~~Bump mapping~~
- ~~Shadows (day/night cycle?)~~
- ~~Look through glass? (first person?)~~
- C̶a̶r̶ ̶t̶u̶r̶n̶i̶n̶g̶ ̶a̶n̶d̶ ̶W̶S̶A̶D̶ ̶c̶o̶n̶t̶r̶o̶l̶s̶
- ~~Car physic collisions to stay on road~~
- ~~Circular buffer of terrain (see terrain.cc constructor)~~
- ~~Car lights (spotlights)~~
- Sounds (car horn? engine, brakes, tight corner screech?)  - Very crude (requires SDL2 change)
- ~~Water (Mitch has prototype)~~
- ~~Fog (Mitch has prototype)~~
- ~~Spotlighting (Andrew has prototype)~~

Low Priority Addons:
- ~~find new loadable car model~~
- lamp lights?
- bushes or something on terrain
- ~~skybox~~
- ~~angle terrain tiles so roads don't connect all funky~~
- ~~x^3 terrain tiles not just x squared~~
- ~~clouds~~
- score system (highscores?)
- birds
- gear shift overlay

**Comment by andrewphamvk** [comment]
TO DO:
Split these TO DOs into separate issues

**Comment by KonradJanica** [comment]
Good idea. Someone pick one and start a new issue for it, that will make it clear who is working on it.

I'll cross these out as they are finished
